microprofile.jdt/org.eclipse.lsp4mp.jdt.tp/org.eclipse.lsp4mp.jdt.target

```diff
@@ -2,7 +2,6 @@
 <?pde version="3.8"?>
 <target name="LSP4MP Target Platform">
 	<locations>
-		<location type="Target" uri="https://repo.eclipse.org/content/repositories/jdtls-snapshots/org/eclipse/jdt/ls/org.eclipse.jdt.ls.tp/1.16.0-SNAPSHOT/org.eclipse.jdt.ls.tp-1.16.0-20220915.034113-10.target"/>
```

Comment:
Could we just comment this out and make the uri `https://raw.githubusercontent.com/eclipse/eclipse.jdt.ls/master/org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target` ? For ease of local development ?